src/libraries/System.Reflection.MetadataLoadContext/tests/src/Tests/Type/TypeTests.StructLayoutAttribute.cs

```diff
@@ -193,6 +194,22 @@ public static void Test_Interface()
             Assert.Null(s);
         }
 
+        [Fact]
+        [ActiveIssue("https://github.com/dotnet/runtime/issues/100220", typeof(PlatformDetection), nameof(PlatformDetection.IsMonoRuntime))]
+        public static void StructWithGCAndExplicitSizeIsAlignedByPtr()
```

Comment:
This test does not belong into this file. These are System.Reflection.MetadataLoadContext tests. They are meant to test MetadataLoadContext projections of the types, not the behavior of the runtime type system.

Reply:
@jkotas any idea what is a more appropriate place? I had a hard time finding the right spot

Reply:
Somewhere under src\tests\Loader\classloader\explicitlayout